endpoints/orders.yaml

```diff
@@ -303,7 +303,7 @@ paths:
                   $ref: 'schemas/commonSchemas.yaml#/components/schemas/subscriptionRequest'
           responses:
             '201':
-              description: Successful.
+              description: Your request was received and an asynchronous response payload will be delivered to the subscriptions counterparty endpoint. Counterparty endpoint URLs use the following endpoint to set URLs - /v1/orders/subscriptions.
```

Comment:
This the endpoint to set the callback URLs, it's synchronous unlike the TVB flow endpoints. Update this description to something like this: "Callback URL set successfully"